From 322592b399d54febf3a87a673b73678363dff1e0 Mon Sep 17 00:00:00 2001
From: jose-isac <joseisac@tutamail.com>
Date: Sun, 27 Apr 2025 14:20:37 -0300
Subject: [PATCH] Habamax color scheme

Habamax, made by Maxim Kim, is one of Vim's default color schemes.
---
 config.def.h | 42 +++++++++++++++++++++---------------------
 1 file changed, 21 insertions(+), 21 deletions(-)

diff --git a/config.def.h b/config.def.h
index 2cd740a..ffaca7a 100644
--- a/config.def.h
+++ b/config.def.h
@@ -93,35 +93,35 @@ char *termname = "st-256color";
  */
 unsigned int tabspaces = 8;
 
-/* Terminal colors (16 first used in escape sequence) */
+/* Habamax terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#1c1c1c",
+	"#af5f5f",
+	"#5faf5f",
+	"#af875f",
+	"#5f87af",
+	"#af87af",
+	"#5f8787",
+	"#9e9e9e",
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#767676",
+	"#d75f87",
+	"#87d787",
+	"#d7af87",
+	"#5fafd7",
+	"#d787d7",
+	"#87afaf",
+	"#bcbcbc",
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
-	"gray90", /* default foreground colour */
-	"black", /* default background colour */
+	"#ffffff", /* cursor background */
+	"#000000", /* cursor foreground */
+	"#bcbcbc", /* foreground colour */
+	"#1c1c1c", /* background colour */
 };
 
 
-- 
2.49.0

